src/test/java/org/opensearch/geospatial/action/upload/geojson/UploaderTests.java

```diff
@@ -173,7 +173,7 @@ public void testBulkActionWithFailedIndexRequest() {
         uploader.upload(content, INDEX_ALREADY_EXIST, mockListener);
         verify(mockBulkRequestBuilder).execute(any(ActionListener.class));
         verify(mockPipelineManager).delete(anyString(), any(StepListener.class), any(Supplier.class));
-        verify(mockListener).onFailure(any());
+        verify(mockListener).onResponse(any());
```

Comment:
Why did this get flipped?

Reply:
In the past, we only return "acknowledged" : true if everything is succeeded, For ex: if we are indexing 100 documents, we only return response if all 100 docs are indexed successfully, else, we will raise an exception with error message from BulkResponse. 
Now, we are following response similar to BulkResponse, where we don't raise exception if any document is failed, instead, we always return response with error message as part of response.

Reply:
The corresponding flipping is here: https://github.com/opensearch-project/geospatial/pull/51/files#diff-08f4455db801d175aa70dfa4ed62818ea8a503ff5f764bb552452c238e99a5bfL123